server/src/main/java/org/elasticsearch/index/mapper/ErrorBehaviour.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.index.mapper;
+
+import java.util.Locale;
+import java.util.Objects;
+
+public enum ErrorBehaviour {
```

Comment:
nit: how about renaming it to OnScriptError?

Reply:
done